CMakeLists.txt

```diff
@@ -150,6 +152,23 @@ if(ROBOTOLOGY_ENABLE_TELEOPERATION)
   find_or_build_package(walking-teleoperation)
 endif()
 
+# Human Dynamics Estimation
+if(ROBOTOLOGY_ENABLE_HUMAN_DYNAMICS)
+  if (WIN32)
+        set(ROBOTOLOGY_ENABLE_ICUB_HEAD TRUE)
+        set(ROBOTOLOGY_USES_CFW2CAN TRUE)
+        set(ROBOTOLOGY_USES_ESDCAN TRUE)
+        find_or_build_package(forcetorque-yarp-devices)
```

Comment:
`forcetorque-yarp-devices` only compiles on Windows?

Reply:
If I am not wrong, only we need it for the producer side using FT sensors. So, that's why I add it for window.

Reply:
Unless the compilation time on Linux/macOS does not explode, I would add it to compilation in any case, so that if any moment we want to start using the FTShoes on Linux (for example for iFeel) we can, and we simplify the superbuild logic as well.